pkg/api/volume.go

```diff
@@ -91,6 +108,7 @@ func (v *VolumePortal) CreateVolume() {
 		Size:              result.Size,
 		AvailabilityZone:  result.AvailabilityZone,
 		ProfileId:         result.ProfileId,
```

Comment:
Please add a TODO item so that we could remove `ProfileId` later.

Reply:
done